### PR TITLE
[azopenai] Enabling live testing

### PR DESCRIPTION
**Pull request by richardpark-msft**
Enabling live testing for both OpenAI and Azure OpenAI endpoints.

### CODE REVIEW

File: sdk/ai/azopenai/ci.yml [hunk 1]
```diff
@@ -26,3 +26,33 @@ stages:
...
+        AOAI_CHAT_COMPLETIONS_MODEL_DEPLOYMENT_CANARY: $(AOAI-CHAT-COMPLETIONS-MODEL-DEPLOYMENT-CANARY)
+        AOAI_ENDPOINT_CANARY: $(AOAI-ENDPOINT-CANARY)
+
+        # OpenAI
+        OPENAI_API_KEY: $(OPENAI-API-KEY)
+        OPENAI_ENDPOINT: $(OPENAI-ENDPOINT)
+        OPENAI_EMBEDDINGS_MODEL: $(OPENAI-EMBEDDINGS-MODEL)
+        OPENAI_CHAT_COMPLETIONS_MODEL: $(OPENAI-CHAT-COMPLETIONS-MODEL)
+        OPENAI_COMPLETIONS_MODEL: $(OPENAI-COMPLETIONS-MODEL)
+
```
Comment (jhendrixMSFT) [inline sdk/ai/azopenai/ci.yml:58]: Nit, missing new-line.

Comment (richardpark-msft) [reply sdk/ai/azopenai/ci.yml]: Fixed.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**